### DISCUSSION TITLE
Problème accès au marketplace du VSCode

### DISCUSSION THREAD

**Discussion by anistouati** [MTESSDev/FRW#280] [Annonces]
Bonjour,

J'essaye d'installer cette [extension](https://marketplace.visualstudio.com/items?itemName=MTESS.vscode-mtess-frw-bacasable)  pour le bac a sable de FRW.

Dans ma VM, je recois ce message : 

![image](https://github.com/user-attachments/assets/98cddd74-26a4-4e0a-911f-b0f1428cd4f1)

C'est le cas pour les autres dev aussi.

**Comment by j0ber** [comment]
Bonjour,

Le message est clair, il faudra voir avec votre organisation afin de vous donner accès au marketplace Visual Studio. Il vous manque peut-être un groupe de sécurité.

C'est assez essentiel pour ceux qui utilisent VSCode.

**Reply by anistouati** [reply]
Merci, on voulais s'assurer avant de passer la demande d'accès.
